salt/config/__init__.py

```diff
@@ -60,6 +67,30 @@
     _DFLT_IPC_MODE = 'ipc'
     _MASTER_TRIES = 1
 
+def _gather_buffer_space():
+    '''
+    Gather some system data and then calculate
+    buffer space.
+
+    Result is in bytes.
+    '''
+    if HAS_PSUTIL:
+        # Oh good, we have psutil. This will be quick.
+        total_mem = psutil.virtual_memory().total
+    else:
+        # We need to load up some grains. This will be slow.
+        os_data = salt.grains.core.os_data()
+        grains = salt.grains.core._memdata(os_data)
+        total_mem = grains['mem_total']
+    # Return the higher number between 5% of the system memory and 100MB
+    return min([total_mem * 0.05, 10 << 20])
```

Comment:
The comment says "Return the higher number", but doesn't `min` return the lower number?